### PR TITLE
Adds multi-task support to Koji scratch builds.

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: setup.rb [hunk 1]
```diff
@@ -219,11 +223,13 @@ def setup_koji_repos(os)
...
 else
-  system('yum -y install katello')
+  system('yum -y install rubygems')
+  system('yum -y install rubygem-kafo')
+  system('yum -y install katello-sam')
 end
 
 installer_options = options[:installer_options] || ""
-install_command = "katello-installer #{installer_options}"
+install_command = "sam-installer #{installer_options}"
```
Comment (dustints) [inline setup.rb]: @ehelms was the change to sam-installer intentional?

**Comment by dustints** [comment]
tested it and looks good except that `sam-katello` not present in nightlies yet.
setup output: https://gist.github.com/5578e412f1643da2529d

**Comment by ehelms** [comment]
Good catch, that was from my testing of SAM with this setup. Updated.

**Comment by jlsherrill** [comment]
ACK from me

**Comment by dustints** [comment]
ACK, looks good.

**Pull request merged**